src/app/(auth)/register/page.tsx

```diff
@@ -19,41 +19,37 @@ export default async function Register({ searchParams }: PropTypes) {
         shouldRedirect: false,
     })
 
+    let userId = user?.id
+    let shouldLogOut = false
+
     if (typeof searchParams.token === 'string') {
         const verify = await verifyUserEmailAction(searchParams.token)
         if (!verify.success) {
-            console.log(verify)
-            return <p>Token er ugyldig</p>
+            redirect('./register')
         }
 
         if (user && verify.data.id !== user.id) {
-            // TODO: Logout
-            console.log('Should logout')
+            // throw new Error('Email is verified. Cannot continue registrations while another user is logged in.')
```

Comment:
Should this error be commented out? If yes then a comment explaining why,and wheter/when it should be uncommented would be nice.